web/vtadmin/src/components/NavRail.tsx

```diff
@@ -17,10 +17,10 @@ import * as React from 'react';
 import { Link, NavLink } from 'react-router-dom';
 
 import style from './NavRail.module.scss';
-import logo from '../img/vitess-icon-color.svg';
 import { useClusters, useGates, useKeyspaces, useSchemas, useTablets, useVtctlds, useWorkflows } from '../hooks/api';
 import { Icon, Icons } from './Icon';
 import { getTableDefinitions } from '../util/tableDefinitions';
+import VitessLogo from './VitessLogo';
```

Comment:
These changes are also the same in OS VTAdmin: https://github.com/vitessio/vitess/pull/9866